src/Export/Adapters/DateAddedAdapter.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace FINDOLOGIC\FinSearch\Export\Adapters;
+
+use FINDOLOGIC\Export\Data\DateAdded;
+use Shopware\Core\Content\Product\ProductEntity;
+
+class DateAddedAdapter
+{
+    public function adapt(ProductEntity $product): ?DateAdded
+    {
+        $releaseDate = $product->getReleaseDate();
+
+        if (!$releaseDate) {
+            return null;
+        }
```

Comment:
This should also work:
```suggestion
        if (!$releaseDate = $product->getReleaseDate()) {
            return null;
        }
```